js/source-map/index.js

```diff
@@ -54,8 +57,30 @@ function getSourcePathAndLineFromSourceMaps (filename, line, column = 0) {
   return { path: filename, line, column }
 }
 
+function getSourcePathAndLineFromSourceMaps (filename, line, column = 0) {
+  const sourceMap = rewrittenSourceMapsCache.get(filename)
+  return getPathAndLine(sourceMap, filename, line, column)
+}
+
+function getOriginalPathAndLine (filename, line, column = 0) {
+  if (filename && line) {
+    try {
+      let sourceMap = originalSourceMapsCache.get(filename)
```

Comment:
We could filter some files checking if the file has been rewritten before: only if `filename` is in `rewrittenSourceMapsCache` continue with the rest of the logic. That will avoid some vulnerability resolution in third-party libs.